metrics/web/docs/features/remote_config/diagrams/remote_config_data_class.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
```

Comment:
```suggestion
@startuml remote_config_data_layer_class_diagram
```
Please apply the suggestion to all diagrams in this PR. Also, notice the filenames.